### DISCUSSION TITLE
Can't access GitHub IP 185.199.108.153

### DISCUSSION THREAD

**Discussion by mkoohafkan** [community/community#74974] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

This is weird, but my agency VPN is specifically unable to access GitHub IP address `185.199.108.153`, i.e. `cdn-185-199-108-153.github.com`which results in GitHub.com and any gh-pages sites not loading.  Other GitHub IPs seem to work fine,  so the result is that GitHub intermittently stops working for a few minutes. This happens frequently throughout the day. We not been able to identify any settings or restrictions in our configuration that would be blocking this address. Any ideas on what might be going on?

**Comment by jankap** [comment]
I'm seeing the same problem. I assume there's some relation to https://containers.dev, and VSCodes remote container stuff.
I've requested to whitelist https://containers.dev, hopefully it will be fixed then.

**Comment by jankap** [comment, marked as answer]
@mkoohafkan 
indeed the solution for us was to whitelist https://containers.dev/ in our proxy.
